### PR TITLE
999 ignore des notifs (webhook sendinblue)

### PR DESCRIPTION
**Pull request by alanlr**

### CODE REVIEW

File: server/src/service/applications.js [hunk 1]
```diff
@@ -132,8 +132,15 @@ const sendApplication = async ({ mailer, query, shouldCheckSecret }) => {
 
       application.to_applicant_message_id = emailCandidat.messageId;
       application.to_applicant_message_status = emailCandidat.accepted.length ? "accepted" : "rejected";
-      application.to_company_message_id = emailCompany.messageId;
-      application.to_company_message_status = emailCompany.accepted.length ? "accepted" : "rejected";
+      if (emailCompany.accepted.length) {
```
Comment (bdavidxyz) [inline server/src/service/applications.js]: emailCompany?.accepted?.length

File: server/src/service/applications.js [hunk 2]
```diff
@@ -344,7 +351,10 @@ const updateApplicationStatus = async ({ payload, mailer }) => {
   const event = payload.event;
 
   let messageType = "application";
-  if (payload.subject.startsWith("Votre candidature chez")) {
+  if (payload.subect.startsWith("Réponse")) {
```
Comment (bdavidxyz) [inline server/src/service/applications.js]: subect ? subject ?

**Review by bdavidxyz — CHANGES_REQUESTED** [review]

File: server/src/service/applications.js (hunk 2 above)
Comment (alanlr) [reply server/src/service/applications.js]: bien vu

File: server/src/service/applications.js (hunk 1 above)
Comment (alanlr) [reply server/src/service/applications.js]: done

**Pull request merged**